src/NzbDrone.Core/MetadataSource/SkyHook/Resource/TMDBResources.cs

```diff
@@ -18,6 +18,13 @@ public class MovieSearchRoot
         public int total_pages { get; set; }
     }
 
+    public class PersonCreditsRoot
+    {
+        public MovieResult[] cast { get; set; }
+        public MovieResult[] crew { get; set; }
+        public int id { get; set; }
+    }
+
     public class MovieResult
```

Comment:
Can you make a subclass like with the Item?

Reply:
yea